mmv1/products/workstations/Workstation.yaml

```diff
@@ -52,6 +62,7 @@ examples:
     name: "workstation_basic"
     min_version: beta
     primary_resource_id: "default"
+    primary_resource_name: "fmt.Sprintf(\"tf-test-workstation-cluster%s\", context[\"random_suffix\"]), fmt.Sprintf(\"tf-test-workstation-cluster-config%s\", context[\"random_suffix\"]), fmt.Sprintf(\"tf-test-workstation%s\", context[\"random_suffix\"])"
     vars:
       workstation_cluster_name: "workstation-cluster"
       workstation_config_name: "workstation-config"
```

Comment:
Fixed, I'm unsure why this is necessary though - especially this naming, as everywhere we use `workstation` instead of `work-station`

Reply:
In general, we _should_ be including hyphens in every variable so that they generate with a `tf-test` prefix and a random suffix (to avoid collisions and make sure we can automatically clean up stray resources). In this particular case, it may not matter, since the workstation is a nested resource and might be cleaned up automatically when its parents (which contain hyphens already) are deleted. I'm not sure offhand.

In the context of this PR, the most important part is just that whatever is in the `primary_resource_name` matches what is generated based on the variables.

Reply:
Check, thanks